config/protocol.mainnet.yml

```diff
@@ -24,7 +24,13 @@ ProtocolConfiguration:
     RegisterTransaction: 10000
 
 ApplicationConfiguration:
-  DataDirectoryPath: "./chains/mainnet"
+  DBConfiguration:
+    DBType: "leveldb" #other options: 'inmemory','redis'
+    DataDirectoryPath: "./chains/mainnet"
+    #Connection settings used in case of Redis
+    Addr: "localhost:6379"
+    Password: ""
+    DB: 0
```

Comment:
Better comment this stuff out, config shouldn't contain ineffective statements, it confuses people, I think.

Reply:
Still there.

Reply:
AM I right you want to comment out all config which are not used right now and in case you need redis you have to uncomment them back? Cause for me it's a bit weird to do more than one action to enable any other DB

Reply:
Yep. IMO having configuration for all DBs active makes it less obvious which one really is used. That's debatable, of course, but for me it's easier to parse active fields to see and understand active configuration (skipping comments) and only look in the comments to see what other options do I have. And if I'm to change the DB used I'll uncomment DB-specific options for new DB and comment out DB-specific options for old DB.

Reply:
ok, done in this way